build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
   
   <PropertyGroup>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
```

Comment:
@nguerrera what targets is it trying to call? i don't see any change related to this property having been made recently.